cli/internal/cmds/neosync/sync/sync.go

```diff
@@ -605,7 +605,15 @@ func syncData(ctx context.Context, cfg *benthosConfigResponse, logger *charmlog.
 		}
 	}()
 
+	split := strings.Split(cfg.Name, ".")
+	var runType string
+	if len(split) != 0 {
+		runType = split[len(split)-1]
+	}
 	streambldr := env.NewStreamBuilder()
+	if outputType == output.PlainOutput {
+		streambldr.SetPrintLogger(logger.With("Benthos", "true", "Table", cfg.Table, "RunType", runType).StandardLog())
```

Comment:
Gonna nit this but I think in the worker we do lower-cased labels instead of upper case.
```suggestion
		streambldr.SetPrintLogger(logger.With("benthos", "true", "table", cfg.Table, "runType", runType).StandardLog())
```

Any specific reason why using uppercase here?